### ISSUE TITLE
TypeError: 'type' object is not subscriptable

### ISSUE THREAD

**Issue by d4rkc0de** [AntonOsika/gpt-engineer#24]
System: Ubuntu 20.04
Python version 3.8

I got this error after running `python main.py example `:

```
Traceback (most recent call last):
  File "main.py", line 7, in <module>
    from ai import AI
  File "/home/user/Projects/trending/gpt-engineer/ai.py", line 5, in <module>
    class AI:
  File "/home/user/Projects/trending/gpt-engineer/ai.py", line 23, in AI
    def next(self, messages: list[dict[str, str]], prompt=None):
TypeError: 'type' object is not subscriptable
```

I already run these commands:

```
pip install -r requirements.txt
export OPENAI_API_KEY=[your api key]
```

In my venv

**Comment by d4rkc0de** [comment]
Fixed by upgrading from Python 3.8 to python 3.10

**Comment by dshahrokhian** [comment]
Happy to see this already resolved on the same day I'm getting this error :))
